src/osa/scripts/gain_selection.py

```diff
@@ -417,8 +417,8 @@ def check_warnings_in_logs(date: datetime, run_id: int) -> bool:
     for file in log_files:
         content = file.read_text().splitlines()
         for line in content:
-            if "WARNING" in line:
-                log.warning(f"There is a warning in the log files of run {run_id}: {line}")
+            if "WARNING - You should use heuristic identification of FF events!" in line:
+                log.warning(f"You should use heuristic identification of FF events for run {run_id}!")
```

Comment:
I wonder in this case, would it be easy to pass this information to the gain selection command builder to add the ff heuristic and run it or prepare it for a second execution with that option?

Reply:
I guess then this would be problematic in subsequent R0 to DL1 processing, since I think you have to be coherent in the usage of FF heuristic option